Tutorials/P5JS/p5.js/09/9.01_p5.js_Transformation_Pt.1/9.01-p5.js-Transformation-Pt.1.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "9.01 p5.js Transformation Pt.1"
+video_number: 9
+date: 2017-09-18
+video_id: o9sgjuh-CBM
+
+contributions:
+  - title: "A visual aid for understanding transformations"
+    author:
+      name: "Adam Nizol"
+    url: "https://adamnizol.github.io/Transformation-Visualization-in-Processing/"
+    source: "https://github.com/AdamNizol/Transformation-Visualization-in-Processing"
+---
```

Comment:
The original description from YouTube should be added here as well so that people who check out the video's page on the website know what it is about.
```suggestion
---

In this three part tutorial, I cover matrix "transformations" using p5.js. In this video I cover the functions `translate()`, `rotate()`, `push()` and `pop()`.
```